src/_pytest/main.py

```diff
@@ -367,6 +365,22 @@ def pytest_runtestloop(session: Session) -> bool:
     return True
 
 
+def _in_build(path: Path) -> bool:
+    """Attempt to detect if ``path`` is the root of a buildsystem's artifacts
```

Comment:
It feels like this isn't just an attempt but a complete piece of logic. So I'd just go with “Detect” since it seems more accurate.
```suggestion
    """Detect if ``path`` is the root of a buildsystem's artifacts
```

Reply:
fixed